Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
      *
      * @param {string|Object} subject The html selector or object on which function should be applied
      */
-    setup_checkbox_range_selection: function(subject) {
-        Admin.log('[core|setup_checkbox_range_selection] configure checkbox range selection on', subject);
+    "setup_checkbox_range_selection" (subject) {
+
+        Admin.log("[core|setup_checkbox_range_selection] configure checkbox range selection on", subject);
 
-        var previousIndex,
+        let previousIndex,
             useICheck = window.SONATA_CONFIG && window.SONATA_CONFIG.USE_ICHECK
         ;
 
         // When a checkbox or an iCheck helper is clicked
-        jQuery('tbody input[type="checkbox"], tbody .iCheck-helper', subject).click(function (event) {
-            var input;
+        jQuery("tbody input[type=\"checkbox\"], tbody .iCheck-helper", subject).click(function (event) {
```

Comment:
- eslint: Line 153 exceeds the maximum line length of 80.
- eslint: Unexpected unnamed function.